packages/paste-design-tokens/tokens/global/sizing.yml

```diff
@@ -49,8 +49,11 @@ props:
     value: "{!size-square-30}"
     comment: Icon sizing token
   size-icon-20:
-    value: "{!size-square-50}"
+    value: "{!size-square-40}"
```

Comment:
Shifting the icon size scale. New default is 20px, shift the last 2 up one in the scale